cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,35 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
-  });
-
-  it('', () => {
+describe('Conduit', () => {
+  const userName = 'nevermore';
+  const email = 'nevermore@gmail.com';
+  const password = 'nevermore123';
+  const title = 'title';
+  const description = 'description';
+  const body = 'body';
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.login(email, userName, password);
   });
 
-  it('', () => {
+  it('should create article with valid data', () => {
+    cy.reload();
+    cy.contains('a', 'New Article').click();
+    cy.get('[placeholder="Article Title"]').type(title);
+    cy.get('[placeholder="What\'s this article about?"]').type(description);
+    cy.get('[placeholder="Write your article (in markdown)"]').type(body);
+    cy.get('.btn-primary').click();
```

Comment:
It's recommended to use data attributes for selecting elements to avoid breaking tests when changing styles or text content. Consider adding data attributes to your elements and use `cy.get('[data-cy=your-data-attribute]')` to select elements.